### DISCUSSION TITLE
Is it possible make compose_build doesn't work?

### DISCUSSION THREAD

**Discussion by JonasTheGamer** [getredash/redash#6873] [Q&A] [answered]
Hi! I tried running the installation instructions for the local development environment today. I couldn't get the make compose_build command to work, so I decided to take a look in the dockerfile.

The error I got was:
 => ERROR [server internal] load metadata for docker.io/library/node:18.1-bookworm

When looking up the image on docker hub, it has to be 18.18.1 instead of 18.1. I changed the dockerfile and ran it again, now the make command runs without any issues.

Do I open an issue for that, or pull request or how does that work? it'd be my first time contributing to a project :-)

**Comment by justinclift** [comment]
@JonasTheGamer Sorry for this.  It does indeed look like a PR merged yesterday broke things. :facepalm:

We'll get it sorted out over the next few hours. :smile:

Btw, if you're into using Discord (?) then [we're often contactable there](https://discord.gg/tN5MdmfGBp) too.

**Comment by justinclift** [comment, marked as answer]
I just pushed a commit to the `master` branch (https://github.com/getredash/redash/commit/960166075183baf0ee8f1d8b88a9eaaa855e3dbc) to get that working again.  Sorry for the problem.

And yeah, that's the kind of thing that's worth opening an issue for.  And/or jumping into Discord as well to ask "wtf?" :smile:

**Comment by JonasTheGamer** [comment]
Aha, thanks! 🤩

**Comment by justinclift** [comment]
Note that there's some mention here that the instructions in the wiki probably need updating after yesterday too: https://github.com/getredash/redash/pull/6876#issuecomment-2050408917

I'm looking at a different issue presently, so I haven't personally checked that at all.

**Comment by JonasTheGamer** [comment]
Alright. Thanks a bunch 😊
